src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -532,7 +522,7 @@ public static SyntaxTree ParseWithRoundTripCheck(string text, CSharpParseOptions
         }
 
         public static CSharpCompilation CreateSubmissionWithExactReferences(
```

Comment:
Looks like this method doesn't match your naming style.

Reply:
I'm holding off on the scripting references for now. They're a bit of a different problem.